queries/gn/highlights.scm

```diff
@@ -0,0 +1,74 @@
+; Includes
+
+"import" @include
+
+; Conditionals
+
+[
+  "if"
+  "else"
+] @conditional
+
+; Repeats
+
+"foreach" @repeat
+
+; Operators
+
+[
+  "="
+  "+="
+  "-="
+  "!"
+  "+"
+  "-"
+  "<"
+  "<="
+  ">"
+  ">="
+  "=="
+  "!="
+  "&&"
+  "||"
+] @operator
+
+; Variables
+
+(identifier) @variable
+
+; Functions
+
+(call_expression function: (identifier) @function.call)
+
+; Fields
+
+(scope_access field: (identifier) @field)
+
+; Literals
+
+(string) @string
+
+(escape_sequence) @string.escape
+
+(expansion) @none
```

Comment:
Is `@none` really needed here considering it's just to nullify highlights, which can be done by not capturing at all? 🤔

Reply:
expansions are potentially inside strings, it's to remove the string hl